lib/ain-rs-exports/src/evm.rs

```diff
@@ -436,6 +436,13 @@ pub fn evm_try_get_block_number_by_hash(
     }
 }
 
+pub fn evm_try_get_block_count(result: &mut ffi::CrossBoundaryResult) -> u64 {
+    match SERVICES.evm.storage.get_latest_block() {
+        Some(block) => cross_boundary_success_return(result, block.header.number.as_u64()),
+        None => cross_boundary_error_return(result, "Unable to get block count"),
+    }
+}
+
```

Comment:
Better to use the exposed function rather than calling storage directly.

Also is it necessary to truncate the response to `u64`? Would be better to return `U256`.

```suggestion
pub fn evm_try_get_block_count(result: &mut ffi::CrossBoundaryResult) -> U256 {
    match SERVICES.evm.block.get_latest_block_hash_and_number() {
        Some((_ , number)) => cross_boundary_success_return(result, number),
        None => cross_boundary_error_return(result, "Unable to get block count"),
    }
}

```